src/index.html

```diff
@@ -21,7 +21,425 @@
     />
```

Comment:
Link to the stylesheet should have an extension `.css` instead of `.scss`. SCSS files are preprocessed into CSS, and the browser only understands CSS files.